Dockerfile

```diff
@@ -16,6 +16,7 @@ ARG branch_sdntrace=master
 ARG branch_kytos_stats=master
 ARG branch_sdntrace_cp=master
 ARG branch_of_multi_table=master
+ARG branch_kafka_events=main
```

Comment:
@Auwate let's take the opportunity to change `kafka_events` default branch as `master`. Understandable that GitHub uses `main` now, but let's keep the same as we have on other NApps to avoid confusion. You have maintainer privilege on `kafka_events` so you should be able to do it, if you need help, let me know.